sfe/pages/unpause-status.html

```diff
@@ -2,26 +2,46 @@
 
 <div class="section">
 
-    {{ if eq .Successful true }}
-
-    <h1>Account successfully unpaused</h1>
+    {{ if and .Successful (gt .Count 0) (lt .Count .Limit) }}
+    <h1>Successfully unpaused all {{ .Count }} identifier(s)</h1>
     <p>
         To obtain a new certificate, re-attempt issuance with your ACME client.
         Future repeated validation failures with no successes will result in
         identifiers being paused again.
     </p>
 
-    {{ else }}
+    {{ else if and .Successful (eq .Count .Limit)}}
+    <h1>Some identifiers were unpaused</h1>
+    <p>
+        We can only unpause a limited number of identifiers for each request ({{
+        .Limit }}). There are potentially more identifiers paused for your
+        account.
+    </p>
+    <p>
+        To attempt to unpause more identifiers, please visit unpause URL from
```

Comment:
to remove reduplication of the word "please", and add an article
```suggestion
        To attempt to unpause more identifiers, visit the unpause URL from
```